Add deviation for explicit BGP RIB streaming config

akalluru1 opened the pull request:
RT-1.30 verifies the BGP RIB state streamed through following YANG path: /network-instances/network-instance/protocols/protocol/bgp/rib

Arista devices need explicit BGP RIB streaming config to send data for the above YANG path.

ram-mac commented on proto/metadata.proto:
```diff
@@ -1044,7 +1044,10 @@ message Metadata {
     bool syslog_non_default_vrf_unsupported = 351;
-    
+
+    // Device that requires explicit config to support BGP RIB streaming
```
@akalluru1 - Can you please create a bug for the deviation for us to track it

ram-mac commented:
@akalluru1 - Also are we sure this deviation is required only for just the nested_policy test? How about the chained_policy_test?

akalluru1 commented:
By mistake discarded the changes in https://github.com/akalluru1/featureprofiles/tree/main which resulted in closing this PR. I've addressed above comments from @ram-mac & created a new PR - https://github.com/openconfig/featureprofiles/pull/4955
